src/gen.rs

```diff
@@ -260,50 +259,48 @@ fn gen_fn_type_for_trait(
 
     // If this requirement is not satisfied, we emit an error.
     if method.is_none() || trait_def.items.len() > 1 {
-        return trait_def.span()
-            .err(
-                "this trait cannot be auto-implemented for Fn-traits (only traits with exactly \
-                 one method and no other items are allowed)"
-            )
-            .emit_with_attr_note();
+        emit_error!(
+            trait_def.span(),
+            "this trait cannot be auto-implemented for Fn-traits (only traits with exactly \
+             one method and no other items are allowed)"
+        );
+        return Err(());
     }
 
     // We checked for `None` above
     let method = method.unwrap();
     let sig = &method.sig;
 
-
     // Check for forbidden modifier of the method
     if let Some(const_token) = sig.constness {
-        return const_token.span()
-            .err(format!(
-                "the trait '{}' cannot be auto-implemented for Fn-traits: const methods are not \
-                    allowed",
-                trait_def.ident,
-            ))
-            .emit_with_attr_note();
+        emit_error!(
+            const_token.span(),
+            "the trait '{}' cannot be auto-implemented for Fn-traits: const methods are not \
+             allowed",
+            trait_def.ident
+        );
```

Comment:
I much prefer to indent a continued string 4 spaces more, instead of 1 like you do here. Would be great if you could change that, also in all other places where this happened.

Reply:
Well, why not?